velox/functions/prestosql/InPredicate.cpp

```diff
@@ -59,10 +59,10 @@ std::optional<std::pair<std::vector<T>, bool>> toValues(
 // Creates a filter for constant values. A null filter means either
 // no values or only null values. The boolean is true if the list is
 // non-empty and consists of nulls only.
-template <typename T>
-std::pair<std::unique_ptr<common::Filter>, bool> createBigintValuesFilter(
+template <typename T, typename U>
+std::pair<std::unique_ptr<common::Filter>, bool> createValuesFilter(
```

Comment:
We are gaining any reuse by changing `createBigintValuesFilter` to `createValuesFilter` since we specialize `int128_t` below.
Let's keep `createBigintValuesFilter` as is and add `createHugeintValuesFilter`

Reply:
Thanks for the suggestion, updated.